### PR TITLE
Add rubies for ppc64le

### PR DESCRIPTION
**Pull request by akhileshbhople**
Added code changes so that travis-rubies can work on ppc64le.

Verified : https://travis-ci.org/akhileshbhople/travis-rubies

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -72,6 +72,21 @@ matrix:
       - sw_vers
     osx_image: xcode9.4 # macOS 10.13
     cache: *osx_cache
+  - os: linux-ppc64le
+    dist: xenial
+    language: generic
```
Comment (joepvd) [inline .travis.yml]: Would be good to get this to build on a more recent image.
To that effect, could you try with `language: shell` , and `group: edge`?

File: .travis.yml [hunk 2]
```diff
@@ -72,6 +72,21 @@ matrix:
...
+    before_install:
+      - |
```
Comment (joepvd) [inline .travis.yml]: This workaround should not be necessary anymore

File: .travis.yml [hunk 3]
```diff
@@ -72,6 +72,21 @@ matrix:
...
+         sudo systemctl stop apt-daily.service &&
+         sudo systemctl kill --kill-who=all apt-daily.service &&
+         while ! (systemctl list-units --all apt-daily.service | fgrep -q dead) ; do
+           sleep 1
+         done
+    before_script:
+      - 'echo -e "gem-wrappers\nrubygems-bundler\nbundler\nrake\nrvm\n" >> ~/.rvm/gemsets/global.gems'
+      - "gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3"
+      - "sudo apt-get install haveged"
```
Comment (joepvd) [inline .travis.yml]: Haveged should already be installed,

File: .travis.yml (hunk 2 above)
Comment (ghatwala) [reply .travis.yml]: latest commit still has this ..hope its fine ?

File: .travis.yml (hunk 3 above)
Comment (ghatwala) [reply .travis.yml]: latest commit still has this ..hope its fine ?

File: .travis.yml (hunk 1 above)
Comment (ghatwala) [reply .travis.yml]: done ,latest commit has these changes !!!

**Review by joepvd — APPROVED** [review]

**Pull request merged**